src/XRP/issued-currency-client.ts

```diff
@@ -564,6 +564,31 @@ export default class IssuedCurrencyClient {
     )
   }
 
+  /**
+   * Re-enables rippling on the trust line between this account (issuing account) and another account.
+   *
+   * @see https://xrpl.org/rippling.html#enabling-disabling-no-ripple
+   *
+   * @param trustLinePeerAccount The X-Address of the account involved in the trust line to re-enable rippling.
+   * @param currencyName The currency of the trust line to re-enable rippling.
```

Comment:
```suggestion
   * @param currencyName The currency of the trust line being re-enabled to ripple.
```

Reply:
lol sorry to be a nit, I just want our docs to at least be grammatically correct

Reply:
hmm that seems fair i suppose